README.md

```diff
@@ -81,6 +81,27 @@ To avoid this problem, use the `--yes` flag to automatically confirm the prompts
 bevy build --yes web
 ```
 
+## Bevy scaffolding
+
+The `bevy new` command lets you easily scaffold a new Bevy project using a custom template or a minimal template provided by [Bevy](https://github.com/TheBevyFlock/bevy_new_minimal).
```

Comment:
```suggestion
The `bevy new` command lets you easily scaffold a new Bevy project using a custom template or a [minimal template provided by Bevy](https://github.com/TheBevyFlock/bevy_new_minimal).
```

I felt the original version was a bit misleading. A link just called [Bevy](#) made me expect it to link to Bevy's website or repository, not a template. I think expanding it to contain "minimal template" makes it a bit more clear!